logic/file.ts

```diff
@@ -220,10 +221,11 @@ export class FileHelper {
 		};
 
 		const metadata = this.app.metadataCache.getFileCache(file) as CachedMetadata | null;
```

Comment:
Is there a reason we can't look at the metadataCache for a file that shouldn't be counted?

"Shouldn't be counted" in the context of this plugin means no character count, word count, or page count. Users may still like to see Created Date, Last Updated Date, or File Size, all of which live in the metadata.

Reply:
Correction: those live in `file.stat`, not the metadata. You were right here, please disregard.

Reply:
Okay, so aside from a couple of CPU cycles checking that there's no metadata for the file, it seems like there wouldn't be any harm in leaving the shouldCountFile check as-is. But either way, this code does prevent tracking dates and file size for files with disallowed extensions. I'll publish my latest changes (based largely on this PR, but with a couple of small differences) and await your feedback. Thanks!